### DISCUSSION TITLE
Add created by and or household tags to recipe

### DISCUSSION THREAD

**Discussion by sbs863** [mealie-recipes/mealie#4340] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I run a server with multiple households. It's great to be able to filter by the household, but it would also be nice to see who created a recipe while viewing the recipe itself. 

### Please provide a concise description of the feature that would resolve your issue.

A simple reference to the user could be added in the title card, I've provided a couple examples. 
![text](https://github.com/user-attachments/assets/aed03585-f0a6-429f-9dc4-ff5a790a1ae5)
![avatar](https://github.com/user-attachments/assets/8ca025cc-405b-4a56-a348-70e7591d570a)

If using the avatar option, it would be nice to use the same reference in the timeline. 
![Screenshot 2024-10-09 at 10-47-28 Cassoulet](https://github.com/user-attachments/assets/49805d65-0574-4924-9eca-bc100ae9c883)

Down the line, these could link to a user's profile so you could see the other recipes that user has added. 

### Please consider and list out some caveats or tradeoffs made in your design decision

None that I can think of. 

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
I agree, having that readably viewable will be nice. I had a similar motivation for this PR: https://github.com/mealie-recipes/mealie/pull/4319
